rewrite-groovy/src/test/java/org/openrewrite/groovy/tree/ClassDeclarationTest.java

```diff
@@ -353,7 +353,7 @@ class A {
 
     @Issue("https://github.com/openrewrite/rewrite/pull/4346")
     @Test
-    @Disabled("Known issue; still need to explore a fix")
+    @ExpectedToFail("Known issue; still need to explore a fix")
```

Comment:
@knutwannheden, I think we could also drop this test out of https://github.com/openrewrite/rewrite/pull/4922 given you've added a new one there that mimics this exactly.

Reply:
Good point! I completely missed that. Which is why I like `@ExpectedToFail`.

Reply:
@timtebeek Thanks for fixing!